src/main/java/com/academy/terai/Controller/ApplicationController.java

```diff
@@ -0,0 +1,59 @@
+package com.academy.terai.Controller;
+
+import com.academy.terai.Model.Application;
+import com.academy.terai.Service.ApplicationService;
+import javassist.NotFoundException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
```

Comment:
Try not to import all classes (don't use * for imports). It literally loads all classes in that package, it's relate heave stuff that should be avoided.